packages/frontend/src/hooks/dashboard/useDashboardFiltersForExplore.ts

```diff
@@ -7,9 +7,9 @@ import { useCallback, useMemo } from 'react';
 import { useDashboardContext } from '../../providers/DashboardProvider';
 
 const useDashboardFiltersForExplore = (
-    tileUuid: string,
+    tileUuid: string | undefined,
     explore: Explore | undefined,
-): DashboardFilters => {
+): DashboardFilters | undefined => {
```

Comment:
Everything else that relies on this works with undefined? I guess so, but I thought I'd ask. That's the only real point of concern here 😄

Reply:
only one was not being caught, done here: https://github.com/lightdash/lightdash/pull/6832/commits/08d1a4ea5707e43a8af81e17196ac71250edec3b